Bugfix wpscan

nigthknight opened the pull request:
If applied this PR will:
- Add wpscan-parser Image build in CI/CD
- Add Example wpscan for old-wordpress
- close #94

nigthknight committed:
Build wpscan-parser Image in CI

nigthknight committed:
Fix Image Name for wpscan-parser

nigthknight committed:
Add Example wpscan for old-wordpress

nigthknight committed:
Change File Extension to match other YAML Files

J12934 commented on .github/workflows/ci.yaml:
```diff
@@ -211,6 +211,17 @@ jobs:
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: scbexperimental/wpscan-parser
```
Would prefer to stick with `scbexperimental/parser-wpscan` to be consistent with the nameing of the other parser images

J12934 commented on scanners/wpscan/values.yaml:
```diff
@@ -1,6 +1,6 @@
 parserImage:
   registry: docker.io
-  repository: scbexperimental/parser-wpscan
+  repository: scbexperimental/wpscan-parser
```
Would prefer to stick with scbexperimental/parser-wpscan to be consistent with the nameing of the other parser images

nigthknight committed:
Change Image Name for Wpscan

To comply with the other Images for Parsers i changed the Name for the
wpscan to parser-wpscan
